kalite/topic_tools/__init__.py

```diff
@@ -523,6 +518,29 @@ def get_assessment_item_data(request, assessment_item_id=None):
     return assessment_item
 
 
+def smart_translate_item_data(item_data):
+    """Auto translate the content fields of a given assessment item data.
+
+    An assessment item doesn't have the same fields; they change
+    depending on the question. Instead of manually specifying the
+    fields to translate, this function loops over all the relevant
+    fields of item_data and translates only the content field.
+    """
+    if 'content' in item_data:
+        item_data['content'] = _(item_data['content'])
+
+    for field in ASSESSMENT_DATA_TRANSLATABLE_FIELDS:
```

Comment:
How did you determine the members of `ASSESSMENT_DATA_TRANSLATABLE_FIELDS`? Can we avoid using a magic constant like this if we just iterate over all the keys in item_data:
```python
        for field, field_data in item_data.iteritems()
            if isinstance(field_data, dict):
                item_data[field] = smart_translate_item_data(field_data)
            elif isinstance(field_data, list):
                item_data[field] = map(smart_translate_item_data, field_data)
```
Or is there something I'm missing? Is it much more expensive to consider all of the keys rather than a limited subset?